### ISSUE TITLE
Apps made in Nativefier are not appearing in Start

### ISSUE THREAD

**Issue by jebeld17** [jiahaog/nativefier#740]
### Description

Nativefier is not saving apps in the Start Menu. Apps are nowhere to be found, either as Nativefier does not alert the user of the install location the first time an app is made.

### Steps to reproduce issue
1. Download and install Node.JS LTS for Windows 10
2. Download and install Nativefier
3. Attempt to create a basic web app - let's use Azure Portal as an example:
'nativefier --name "Azure Portal" "http://portal.azure.com"'
4. App downloads and installs without errors. No sign of the app anywhere in the Start panel or Desktop.

### Details

- Are you nativefying a *public* website?
Yes
- Feature request? Have you looked at `nativefier --help` to see if an existing option could fit your needs?
No
- Full nativefier command used to build your app: `<command here>`
See above
- Version of Nativefier (run `nativefier --version`): `v0.0.0`
V 7.6.10
- Version of node.js (run `node --version`): `v0.0.0`
v 10.15.0 LTS
- OS: `<OS here>`
Windows 10 17134.523 (Semi-Annual Channel Non-Targeted Update Ring)
- Error message / stack trace (if any):
n/a
<Error message / Stack trace here>
n/a

**Comment by jebeld17** [comment]
#335

**Comment by githubbob42** [comment]
Perhaps I'm not understanding your issue, but Nativefier doesn't "download and install an app" of the website you specify.  Nativefier builds an Electron executable that simply displays the website you pointed to (look in the `build` folder that gets created).  I think you may be confusing `App downloads and installs without errors` with the app built an Electron executable (again - look in the `build` folder where you ran the `nativefier` command).

If you want the resulting electron executable to be "installable", you'll need to take the executable (and associated files) and build an installation package with an install builder tool (e.g. [WIX Toolset](http://wixtoolset.org/), etc).  

Basically, Nativefier only creates a wrapper application around a chromium browser, pointing to the website you defined at build time.

**Comment by jebeld17** [comment]
@githubbob42, what I meant by "Download and Install" is actually "Download and Save."
Sorry for the confusion. I am updating this bug to reflect this.

**Comment by ronjouch** [comment]
You want nativefier to create a Start Menu shortcut automatically. It's being done for Linux in https://github.com/jiahaog/nativefier/pull/729 , PR welcome for a similar feature on Windows.

**Comment by jebeld17** [comment]
@ronjouch sorry for the confusion again. I updated this bug to reflect the underlying issue - Nativefier is not saving the Nativefie-d apps in the default save location.
I will be making a separate ticket for the feature request at a later time and date. Can you please re-schedule this as a bug? :) Thank you! 👍

**Comment by githubbob42** [comment]
@jebeld17 It's been almost a year since I last rebuilt my application so forgive me I sound confused (perhaps new features have added that I'm not aware of)... 

So after you run `nativefier --name "Azure Portal" "http://portal.azure.com"`, what do you expect to happen or the result to be?

**Comment by jebeld17** [comment]
@githubbob42 apologies I am not writing these as in-line replies. Site keeps freezing my browser window when I open a reply prompt.

What I am expecting is Nativefier to build and create a directory or ZIP under the default save location (<user>/downloads) containing a "native" web app executable labeled 'Azure Portal' I can then run from there.

**Comment by githubbob42** [comment]
Thanks.  Assuming we are still on the same page (the output of running the `nativefier` command), I'm not familiar with `/downloads` being the default save location.  

My experience is that `nativefier` builds the executable in a folder, inside the `./build` folder, where you ran `nativefier`.  

So if you ran `nativefier --name "Azure Portal" "http://portal.azure.com"` from the command-line here:
`/Users/you/projects/azurePortal` 

I would expect it to build the executable here:
 `/Users/you/projects/azurePortal/build/Azure_Portal-win32-x64` _(or something similar to that; depending on your platform and options specified)_
